src/BloomExe/ImageProcessing/ImageServer.cs

```diff
@@ -75,20 +77,33 @@ protected override bool ProcessRequest(IRequestInfo info)
 				return false;
 
 			var r = GetLocalPathWithoutQuery(info);
-			if (r.EndsWith(".png") || r.EndsWith(".jpg"))
+
+			// only process these image types
+			if (!r.EndsWith(".png") && !r.EndsWith(".jpg") && !r.EndsWith(".svg")) return false;
+
+			r = r.Replace("thumbnail", "");
+
+			// This happens with the new way we are serving css files
+			if (!File.Exists(r))
```

Comment:
So the essential change here (apart from also finding svgs) is that we look for images anywhere in BloomBrowserUI?